install/kubernetes/README.md

```diff
@@ -1,6 +1,6 @@
 # Install Istio on an existing Kubernetes cluster
 
-Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/quick-start.html).
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/).
 
-If you prefer to install Istio by checking out the [istio/istio](https://github.com/istio/istio) repository, you can run `updateVersion.sh`
-in the parent directory to generate the required installation files.
+Use the [helm installation instructions](https://istio.io/docs/setup/kubernetes/install/helm/)
+if you want to install Istio by checking out the istio/istio repository, istead of downloading a release.
```

Comment:
I don't think we should recommend users install from the repo. We see a lot of github issues from this, because people are using master-latest-daily images (what we have in the repo) or other issues.

Reply:
I agree we don't want to recommend it, but this is in response to [this doc defect](https://github.com/istio/istio.io/issues/4160). Seems to me that we need to say something. I added a similar message in the quick-start doc. Maybe in both places we should add that this is not recommended, like I said in my comment in the issue?

Any other suggestions?

Reply:
Users are going to install from the repo whether we recommend it or not.  We might as well accept that as a fact.  Even developers get wrapped up in the master-latest-daily problem.  I think a better solution to the master-latest-daily problem would be to add instructions in this README indicating "If installing from the repository, please override your docker hub and docker tag as follows..."  And potentially add them on istio.io as a subpage "For developers"...  I know the user docs are user facing, but I'd really like to see some docs for devs - even a one pager, in the main istio site because the wiki is hard to find and not well maintained.

THoughts?

Cheers
-steve

Reply:
good idea, I redirected developers to the wiki